src/App.tsx

```diff
@@ -1,24 +1,79 @@
 import React from 'react';
+// import { render } from 'react-dom';
 import './App.scss';
 
-const goodsFromServer: string[] = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
-
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  goodsFromServer: string[],
+  selectedGood: string[],
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    goodsFromServer: [
+      'Dumplings',
+      'Carrot',
+      'Eggs',
+      'Ice cream',
+      'Apple',
+      'Bread',
+      'Fish',
+      'Honey',
+      'Jam',
+      'Garlic'],
```

Comment:
i don't think this the way to do it. you will be receiving data from server, and it doesn't have to be in your state, as that list does not change at all.